acceptance/tests/windows/env_windows_installdir_fact.rb

```diff
@@ -6,13 +6,20 @@
   confine :to, :platform => 'windows'
 
   agents.each do |agent|
-    step "test for presence/accurance of fact on #{agent}" do
+    step "test for presence/accuracy of fact on #{agent}" do
       platform = agent[:platform]
       ruby_arch = agent[:ruby_arch] || 'x86' # ruby_arch defaults to x86 if nil
 
+      # The output of Beaker's assert match will have escaped backslashes '\'
+      # in facts output, so the output we match against looks like
+      # "C:\\\\Program Files...". However, we can't just supply four slashes
+      # in our string argument to assert_match, because they're escaped
+      # resulting in asserting a match for "C:\\Program Files', which fails.
+      # So we double escape '\\' (eight), resulting in asserting a match for
+      # "C:\\\\Program Files...".
       install_dir = platform =~ /-64$/ && ruby_arch == 'x86' ?
-        "C:\\\\Program Files (x86)\\\\Puppet Labs\\\\Puppet" :
-        "C:\\\\Program Files\\\\Puppet Labs\\\\Puppet"
```

Comment:
So the value coming back from `puppet facts` is `console` format (which could be explicitly specified with `--render-as console` - but it's also just a pretty printed JSON.

Maybe it's better to just do a `JSON.parse` on the string that comes back, and lookup `env_windows_installdir` that way?